ModelVale/ModelVale/CustomComponents/HealthBarView.h

```diff
@@ -0,0 +1,16 @@
+//
+//  HealthBar.h
+//  ModelVale
+//
+//  Created by Chaytan Inman on 7/14/22.
+//
+
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface HealthBarView : UIView
```

Comment:
typically i would like to see this new view in its own PR since its separate from the other code added here

Reply:
completely agree but I'm not sure how to go about fixing that without creating a giant mess, if you or david have time today or tomorrow would you be able to help me?

Reply:
If you are still using sourcetree you can copy this file and save it somewhere (.h and .m), revert the addition of this file, commit the change, create a new branch and then re-add the file. I can explain more tomorrow if you need help